### PR TITLE
Add some basic usage instruction and doc about perf dependency.

### LINKED ISSUE #47: Add documentation in README about how to use inferno
Add some instruction using perf to output a perf record and use inferno to output an svg out of it.

### PR DESCRIPTION
**Pull request by meven**
Fixes #47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,23 @@ It is developed in part through live coding sessions, which you can find
 [on YouTube](https://www.youtube.com/c/JonGjengset). The first video in
 the sequence is [here](https://www.youtube.com/watch?v=jTpK-bNZiA4).
 
+# Dependency
+
+  You need to have the `perf` tool installed on your Linux systems.
```
Comment (jonhoo) [inline README.md]: I think I'd prefer to not have these lines indented.

Comment (jonhoo) [inline README.md]: Could we also link `perf` to https://perf.wiki.kernel.org/index.php/Main_Page?

File: README.md [hunk 2]
```diff
@@ -18,6 +18,23 @@ It is developed in part through live coding sessions, which you can find
...
+  This can involve package similar to `linux-tools-generic` for Ubuntu, `linux-tools` for Debian.
+  You may need to tweak a kernel config such as `echo 0 | sudo tee /proc/sys/kernel/perf_event_paranoid`.
```
Comment (jonhoo) [inline README.md]: Can we link to https://unix.stackexchange.com/q/14227 for why this is necessary?

File: README.md [hunk 3]
```diff
@@ -18,6 +18,23 @@ It is developed in part through live coding sessions, which you can find
...
+
+# How to Use
+
+  Build Inferno
+    `cargo build --release`
+
+  Run a program using profiling with perf
+    `perf record -g -F 99 -a [your program]`
```
Comment (jonhoo) [inline README.md]: I'd cut the `-F 99` part of this. I also don't think you need `-a`?

File: README.md [hunk 4]
```diff
@@ -18,6 +18,23 @@ It is developed in part through live coding sessions, which you can find
...
+# Dependency
+
+  You need to have the `perf` tool installed on your Linux systems.
+  This can involve package similar to `linux-tools-generic` for Ubuntu, `linux-tools` for Debian.
+  You may need to tweak a kernel config such as `echo 0 | sudo tee /proc/sys/kernel/perf_event_paranoid`.
+
+# How to Use
+
+  Build Inferno
+    `cargo build --release`
```
Comment (jonhoo) [inline README.md]: I'd prefer for these to be code blocks rather than inline using <tt>```console</tt>

**Comment by meven** [comment]
Thank you for your suggestions.

**Comment by jonhoo** [comment]
One last thing and then I think we're good to merge: can you use <tt>```console</tt> instead of just <tt>```</tt>, and then start the command with `$ `. That gives slightly better syntax highlighting for commands on GitHub :)

**Comment by meven** [comment]
Thanks again and now the | are highlighted :)
You can have a preview there : https://github.com/meven/inferno/tree/add-usage

**Comment by jonhoo** [comment]
Awesome, thank you!

**Pull request merged**